src/components/switch/switch.tsx

```diff
@@ -1,26 +1,27 @@
-import './switch.scss';
-import { useTheme } from 'styled-components';
-import { useEffect, useState } from 'react';
-import clsx from 'clsx';
-import { getLocalStorageItem } from '../../common/utils/parsedLocalStorage';
+import "./switch.scss";
+import { useTheme } from "styled-components";
+import { useEffect, useState } from "react";
+import clsx from "clsx";
+import { getLocalStorageItem } from "../../common/utils/parsedLocalStorage";
 
-const Switch = ({onChange}) => {
+const Switch = ({ onChange }: { onChange: (state: any | null) => void }) => {
   const theme = useTheme();
 
-  const [checked, setChecked] = useState(undefined);
+  const [checked, setChecked] = useState<boolean | undefined>(undefined);
```

Comment:
Why use `true`, `false` and `undefined`, when you can just use `light` and `dark`?
So much complexity in so simple thing.

Reply:
Alright I will refactor this myself